tests/e2e/nodemapper.go

```diff
@@ -0,0 +1,295 @@
+/*
+Copyright 2018 The Kubernetes Authors.
```

Comment:
since this is a new file, change the copyright year from 2018 to 2020

Reply:
Done